src/interface/src/app/plan/area-notes/area-notes.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component } from '@angular/core';
+
+interface Note {
+  message: string;
+  name: string;
+  date: string;
+}
+
+@Component({
+  selector: 'app-area-notes',
+  templateUrl: './area-notes.component.html',
+  styleUrls: ['./area-notes.component.scss'],
+})
+export class AreaNotesComponent {
+  notes: Note[] = [
```

Comment:
is this just meant to be sample data? Could we put it in a static file or in the test spec?

Reply:
this is just a placeholder, will go away once we plug in the actual rest call

Reply:
I could probably just delete this, actually, as you can add notes and just see them pop im.  I'll do that.